### PR TITLE
update fedora release howto

### PR DESCRIPTION
**Pull request by jchapma**

### CODE REVIEW

File: docs/389ds/howto/howto-fedora-release-process.md [hunk 1]
```diff
@@ -4,111 +4,124 @@ title: "Fedora Release Process"
...
-- Edit the **/tmp/cl-info** file. Remove the hash prefix value for all bugzilla and github issues. Leave the hash for coverity/misc updates.
 
 ### **Fedora** - Dist-Git - New release
+In general, when we do a new build, we push changes to rawhide first, so that the development branch has the newest bits, then to active Fedora releases ( f39, f40)
+
+- Checkout the source
 
-    git checkout rawhide  (or f39, f38, ...)
+		cd /home/$USER/source/fedora/389-ds-base
+		git checkout f39
```
Comment (mreynolds389) [inline docs/389ds/howto/howto-fedora-release-process.md:65]: why such huge indentation?

File: docs/389ds/howto/howto-fedora-release-process.md [hunk 2]
```diff
@@ -4,111 +4,124 @@ title: "Fedora Release Process"
...
 
 - Update Fedora spec file with Rust packages data 
 
-        DS_SPECFILE=/home/mareynol/source/FEDORA/389-ds-base/389-ds-base.spec make -f rpm.mk bundle-rust
+        DS_SPECFILE=/home/$USER/source/fedora/389-ds-base/389-ds-base.spec make -f rpm.mk bundle-rust
         
     - On 1.4.3:
 
-            FEDORA_SPECFILE=/home/mareynol/source/FEDORA/389-ds-base/389-ds-base.spec make -f rpm.mk bundle-rust-on-fedora
+			FEDORA_SPECFILE=/home/$USER/source/fedora/389-ds-base/389-ds-base.spec make -f rpm.mk bundle-rust-on-fedora
```
Comment (mreynolds389) [inline docs/389ds/howto/howto-fedora-release-process.md:77]: Again, why such a large indent?

**Review by mreynolds389 — APPROVED** [review]
minor comments, but ack from me

**Comment by jchapma** [comment]
> minor comments, but ack from me

I am using ghostwriter to display the markdown output, with this tool I needed these indentations to display the "code" boxes correctly. I agree with you though, they seem extreme. Let me look into using something else.

**Pull request merged**